docs/releases/contributors.rst

```diff
@@ -0,0 +1,29 @@
+
+Contributors
+------------
+
+This page lists all people who have contributed to the development of
+``datatable``. We take into account both code and documentation contributions,
+as well as contributions in the form of bug reports and feature requests.
+
+More specifically, a *code contribution* is considered any PR (pull request)
+that was merged into the codebase. The "complexity" of the PR is not taken
+into account as it is highly subjective. Next, an *issue contribution* is
+any closed issue except for those that are tagged as "question", "wont-fix"
+or "cannot-reproduce". Issues are attributed according to their closing date,
+not their creation date.
+
+In the table, the contributors are sorted according to their total contribution
+score, which is the weighted sum of the count of each user's code and issue
+contributions. Code contributions have more weight than issue contributions,
+and more recent contributions more weight than the older ones.
```

Comment:
more recent contributions *have* more weight than the older ones?

Reply:
This is called [VP ellipsis](https://en.wikipedia.org/wiki/Verb_phrase_ellipsis) -- a perfectly normal grammatical construct in English.